setup.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 setup_reqs = ["setuptools_scm"]
-install_reqs = ["pyserial"]
+install_reqs = [""]
```

Comment:
I'm pretty sure this should be an empty list rather than a list containing a blank string.